server/service/service.go

```diff
@@ -0,0 +1,95 @@
+package service
+
+import (
+	"context"
+
+	"github.com/openconfig/bootz/proto/bootz"
+	"github.com/openconfig/gnmi/errlist"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type EntityLookup struct {
+	Manufacturer string
+	SerialNumber string
+}
+
+type ChassisEntity struct {
+	BootMode string
```

Comment:
It looks like you use this as an enumerated type -- I'd suggest to define a type to avoid magic string matching.